ficlatte/settings-example.py

```diff
@@ -1,106 +0,0 @@
-"""
-Django settings for ficlatte project.
```

Comment:
Be careful of which changes you stage - I'm assuming you didn't mean to delete this file.